Source/Core/VideoBackends/D3D/VertexShaderCache.h

```diff
@@ -51,11 +62,49 @@ class VertexShaderCache
       SAFE_RELEASE(bytecode);
     }
   };
+
+  class VertexShaderCompilerWorkItem : public VideoCommon::AsyncShaderCompiler::WorkItem
```

Comment:
Feels like these two could be the same class (haven't looked at the other backends to see if there's commonality here), they only seem to differ by a single function which returns the shader bytecode.